core/src/main/java/org/apache/calcite/sql/SqlKind.java

```diff
@@ -721,6 +721,26 @@
    */
   MULTISET_QUERY_CONSTRUCTOR,
 
+  /**
+   * The JSON value expression.
+   */
+  JSON_VALUE_EXPRESSION,
+
+  /**
+   * The JSON API common syntax.
+   */
+  JSON_API_COMMON_SYNTAX,
+
+  /**
+   *
```

Comment:
Seems like a comment is missing here (and below)

Reply:
Done, thanks!